src/main/java/com/laborete/LaboreteAPI/constants/ConstantsHelper.java

```diff
@@ -0,0 +1,11 @@
+package com.laborete.LaboreteAPI.constants;
+
+public class ConstantsHelper {
```

Comment:
Personally, I place constants like `firstNameIsRequired` and similar ones to the service they are used in and make them private.